code/game/machinery/jukebox.dm

```diff
@@ -37,7 +37,12 @@ datum/track/New(var/title_name, var/audio)
 		new/datum/track("Thunderdome", 'sound/music/THUNDERDOME.ogg'),
 		new/datum/track("Space Oddity", 'sound/music/space_oddity.ogg'),
 		new/datum/track("Space Asshole", 'sound/music/space_asshole.ogg'),
-		new/datum/track("Velvet Rose", 'sound/music/Velvet-Rose.ogg')
+		new/datum/track("Velvet Rose", 'sound/music/Velvet-Rose.ogg'),
+		new/datum/track("Boolean Sisters", 'sound/music/recordplayer/BooleanSisters.ogg'),
+		new/datum/track("Posin'", 'sound/music/recordplayer/Posin.ogg'),
+		new/datum/track("Jazz Instrumental", 'sound/music/recordplayer/JazzInstrumental.ogg'),
+		new/datum/track("Le Swing", 'sound/music/recordplayer/LeSwing.ogg'),
+		new/datum/track("Cosmorot", 'sound/music/recordplayer/Cosmorot.ogg')
```

Comment:
these are meant to not be in the jukebox, they're in the phonograph

Reply:
I did that since I put the jukebox in the diner and the phonograph in the game room, and I figured people would be able to want to listen to those swing songs in the diner as well.  Should I just put both in each area?  What happens if someone activates both the jukebox and the phonograph at the same time?

Reply:
dunno, only one way to find out, really

Reply:
Turns out turning on the phonograph turns off the jukebox and vice versa.  So both it will be : D

Reply:
...Actually, it doesn't turn off the previous song being played, it just starts a new one.  You can still press "Stop" on the previous music playing machine and it will turn off the music from the new music playing machine.